app/src/main/res/layout/see_all_item_view.xml

```diff
@@ -1,10 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Button style="@style/Widget.AppCompat.Button.Borderless.Colored"
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There aren't any attributes set in this `FrameLayout`. Is there a reason you have wrapped the button in the frame?

Reply:
the button takes up limited width and is centered inside the FrameLayout

Reply:
Cool. I was wondering if that can also be achieved by setting the width of the button to `wrap_content` without wrapping it in a container.

Reply:
Nvm, my brain gave out for a moment.